frontend/src/app/components/MainInstallButton/index.js

```diff
@@ -123,13 +129,11 @@ export default class MainInstallButton extends React.Component {
     }
 
     const makeInstallButton = (extraClass = "") => {
+      const isEnabling = (experimentTitle && hasAddon && !isExperimentEnabled(experiment) && isInstalling);
       return <button onClick={e => this.install(e)}
         className={classnames(`button primary main-install__button ${extraClass}`, { "state-change": isInstalling })}>
         {!isInstalling && installButton}
-        {isInstalling &&
-          <Localized id="landingInstallingButton">
-            <span className="progress-btn-msg">Installing...</span>
-          </Localized>}
+        {isInstalling ? isEnabling ? enablingButton : installingButton : null} {/* eslint-disable-line no-nested-ternary */}
```

Comment:
This seems like a reasonable case to make an exception for the nested ternary rule.